### PR TITLE
Print URI of created repo

### LINKED ISSUE #2: Create doesn't print the URI for the repo it just created
`k8ecr create foo` should print the URI for the repo like `accno.dkr.ecr.eu-west-1.amazonaws.com/foo`

### PR DESCRIPTION
**Pull request by plumdog**
To fix #2.

### CODE REVIEW

File: create.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ type PolicyDocument struct {
 	Statement []StatementEntry
 }
 
-func createRepository(name string) error {
+func createRepository(name string) (ecr.Repository, error) {
```
Comment (winjer) [inline create.go]: return &ecr.Repository, not just a straight ecr.Repository so...

File: create.go [hunk 2]
```diff
@@ -85,27 +85,40 @@ func createRepository(name string) error {
 			},
 		},
 	}
+
+	var repo ecr.Repository
+
 	b, err := json.Marshal(&policy)
 	if err != nil {
-		return err
+		return repo, err
```
Comment (winjer) [inline create.go]: this can then return `nil, err`

File: create.go [hunk 3]
```diff
@@ -85,27 +85,40 @@ func createRepository(name string) error {
...
 	}
 
 	svc := ecr.New(createSession())
-	svc.CreateRepository(&ecr.CreateRepositoryInput{
+	resp, createErr := svc.CreateRepository(&ecr.CreateRepositoryInput{
 		RepositoryName: &name,
 	})
+
+	if createErr != nil {
+		return repo, createErr
```
Comment (winjer) [inline create.go]: and this

**Review by winjer — CHANGES_REQUESTED** [review]
best to never return something uninitialized

**Comment by plumdog** [comment]
Fixed.

And I think I learnt some go.

**Review by winjer — APPROVED** [review]

**Pull request merged**